mne/transforms.py

```diff
@@ -901,13 +901,12 @@ class _TPSWarp(object):
 
     Notes
     -----
-    Adapted from code by `Wang Lin <wanglin193@hotmail.com>`_.
+    Based on the method by :footcite:`Bookstein1989` and,
+    adapted from code by `Wang Lin <wanglin193@hotmail.com>`_.
```

Comment:
I am sorry that I didn't say that before ... but why is 
```
`Wang Lin <wanglin193@hotmail.com>`_.
```
formatted as a website? :)

maybe this would be better

```suggestion
    Based on the method by :footcite:`Bookstein1989` and
    adapted from code by Wang Lin (wanglin193@hotmail.com).
```